### PR TITLE
Attempt to update false to true

### PR DESCRIPTION
**Pull request by dolivas644**
Need help writing put request
First time attempting put request

### CODE REVIEW

**Comment by dolivas644** [comment]
Working Query: 
UPDATE user_anime_list
SET complete = true
WHERE id=$1

**Comment by dolivas644** [comment]
WORKING
the Complete button is clicked column complete value changes freom false to true

File: server/routes/animeListItem.js [hunk 1]
```diff
@@ -69,4 +69,20 @@ const animeItem = req.params.id;
...
+  try {
+    await db.none(`UPDATE user_anime_list SET complete = true WHERE id=$1`, [animeItem]);
+    res.send({status:'success'});
+  } catch (e) {
+    console.log(e);
+    return res.status(400).json(e);
+  }
+})
+
 export default router;
```
Comment (Julzerator) [inline server/routes/animeListItem.js:88]: Don't forget your new line at the end of files!

Comment (dolivas644) [reply server/routes/animeListItem.js:88]: Can you please clarify this?

Comment (cdm-ium) [reply server/routes/animeListItem.js:88]: This is a comment eslint style rule: https://airbnb.io/javascript/#whitespace--newline-at-end. Google around to see why this is a recommendation.

Probably not the most important thing right now unless you are going to set up some automated linting, but it is a good thing to be aware of.

**Pull request merged**